docker/dev-entrypoint.sh

```diff
@@ -9,12 +9,11 @@ APP_DEV_DIR="/home/$USER/ondemand/dev"
 OOD_DEV_DIR="/var/www/ood/apps/dev/$USER"
 
 sudo su root <<SETUP
-  mkdir -p $OOD_DEV_DIR 
-  chmod 755 $OOD_DEV_DIR
+  mkdir -p $OOD_DEV_DIR
   cd $OOD_DEV_DIR
   ln -s $APP_DEV_DIR gateway
 
-  /opt/ood/ood-portal-generator/sbin/update_ood_portal
+  /opt/ood/ood-portal-generator/sbin/update_ood_portal --force
 
   if [ -n "$OOD_STATIC_USER" ] && [ -f "$OOD_STATIC_USER" ]; then
```

Comment:
If we could define a custom static user with ood-portal-generator, this extra logic could be removed.  It's not great in my opinion to have this file managed by ood-portal-generator but then due to lack of flexibility, we completely overwrite it.

Reply:
I think I added this `--force` here because we mount in `/opt/etc/ood/config` so the sha file doesn't exist the very first time you run.

`--force` may actually suite this use case though. We persist the `ood_portal.yml` on the host file system, mounted into the container. And keep the `ood-portal.conf` ephemeral, regenerated every time we start the container up, so we actually don't care what it's previous state was.

Maybe `--rpm` is the right flag here? But I'm happy to force here in this edge case, and keep the `update_ood_portal` the way it is, if nothing else, to just keep the attack surface small.